libafl_frida/src/helper.rs

```diff
@@ -216,6 +218,30 @@ impl<'a> FridaInstrumentationHelper<'a> {
         _harness_module_name: &str,
         modules_to_instrument: &'a [PathBuf],
     ) -> Self {
+        // workaround frida's frida-gum-allocate-near bug:
```

Comment:
Maybe add a reference to the bug / bugtracker, then we know when we can get rid of the workaround?